### PR TITLE
Updating the SIG Apps goals to reflect current state

### PR DESCRIPTION
**Pull request by mattfarina**
/cc @prydonius @michelleN 

While prepping for the SIG Apps update at Kubecon I noticed the goals were a bit dated and could use a little grammar love.

### CODE REVIEW

File: sig-apps/README.md [hunk 1]
```diff
@@ -27,24 +27,27 @@ Meeting recordings can be found [here](https://www.youtube.com/watch?v=hn23Z-vL_
...
+* Do not pick which apps to run on top of Kubernetes
+* Do not recommend one way to do things (e.g., picking a template language)
 
 ## [Helm](https://helm.sh) and [Charts](https://github.com/kubernetes/charts)
+
 Helm and Charts each have their own regular meetings.
 
 * The Helm Developer call is [Thursday at 16:15 UTC](https://zoom.us/j/4526666954). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=16:15&tz=UTC). Meeting notes and agenda can be found [here](https://docs.google.com/document/d/1elWRfvH3AkHdr8pOaqyPbqSZ6ONR-l1Sb9_gapqh8ZA/edit).
-* Charts Chat is [Thursday at 18:00 UTC](https://zoom.us/j/166909412). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=16:15&tz=UTC). Meeting notes and agenda can be found [here](https://docs.google.com/document/d/1h6UTTuNRbFI81higrN3JUV2XxyzqqVjZET4Xz4WTR-8/edit#heading=h.57pbxthvt6k3).
+* Charts Chat is every other [Thursday at 18:00 UTC](https://zoom.us/j/166909412). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=16:15&tz=UTC). Meeting notes and agenda can be found [here](https://docs.google.com/document/d/1h6UTTuNRbFI81higrN3JUV2XxyzqqVjZET4Xz4WTR-8/edit#heading=h.57pbxthvt6k3).
```
Comment (prydonius) [inline sig-apps/README.md]: The day and time is likely to move, but we can update this at a later stage. FWIW, right now it looks like we'll move to Tuesdays at 9AM PT. I thought we were also moving away from specifying times in UTC?

**Comment by mattfarina** [comment]
@prydonius When we move the charts meeting we can update it.

I updated the time on the helm and charts meeting to be based on PT and fixed the timezone conversion links.

**Comment by prydonius** [comment]
thanks
/lgtm

**Pull request merged**